app/src/androidTest/java/com/github/h3lp3rs/h3lp/mainpage/MainPageTestActivity.kt

```diff
@@ -152,17 +149,17 @@ class MainPageTestActivity : H3lpAppTest() {
             )
         }
     }
-/*
-    @Test
-    fun clickingOnProfileButtonWorksAndSendsIntent() {
-        launchAndDo {
-            clickingOnButtonWorksAndSendsIntent(
-                MedicalCardActivity::class.java,
-                withId(R.id.button_profile),
-                false
-            )
-        }
-    }*/
+
+//    @Test
+//    fun clickingOnProfileButtonWorksAndSendsIntent() {
+//        launchAndDo {
+//            clickingOnButtonWorksAndSendsIntent(
+//                MedicalCardActivity::class.java,
+//                withId(R.id.button_profile),
+//                false
+//            )
+//        }
+//    }
```

Comment:
Beats me, it actually already was commented before my PR but I tried to uncomment it, it worked locally but sure enough, it didn't work on Cirrus. I'll actually remove it since there's no real use for it apart from showing that we actually had a test method.